esphome/components/spi_led_strip/spi_led_strip.h

```diff
@@ -0,0 +1,91 @@
+#pragma once
+
+#include "esphome/core/component.h"
```

Comment:
I believe you need to add `defines.h` to ensure `SPIRGBLED_DATA_RATE` is picked up. [See also.](https://github.com/esphome/esphome/pull/5344)

```suggestion
#include "esphome/core/component.h"
#include "esphome/core/defines.h"
```

Reply:
Is `defines.h` something new? Because it worked as it was.

But now that you have merged the new SPI PR, that won't be necessary. I'll update it to dynamically set the rate rather than using the #define kludge.

Reply:
> Is `defines.h` something new? Because it worked as it was.

It'll likely work because many other components include it; it's where any defines added to the project are written by `cg.add_define`. If by some chance nothing includes it, the macro wouldn't work. 😇

> But now that you have merged the new SPI PR, that won't be necessary. I'll update it to dynamically set the rate rather than using the #define kludge.

Sounds great! I'll leave you to it, then... 😁

Reply:
Also, feel free to nuke my last two commits -- was just trying to resolve a merge conflict and I fudged up the codeowners. 😅

Reply:
Ok pending the CI run, should be good to go. Tested here with the changes.

Reply:
Awesome! I'll give it another test on my FunHouse board 😄